lib/text.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+require_relative '../.env.rb'
+
+class Text
+
+  attr_reader :customer_number
+
+  def self.make_text (customer_number, message )
+    our_number = '+441484906157'
+    account_sid = ENV[:account_sid]
+    auth_token = ENV[:auth_token]
+    client = Twilio::REST::Client.new account_sid, auth_token
+    client.account.messages.create({
+    	:from => our_number,
+    	:to => customer_number,
+    	:body => message,
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.